src/Text/Pandoc/Parsing.hs

```diff
@@ -260,6 +261,17 @@ anyLine = do
 anyLineNewline :: Stream [Char] m Char => ParserT [Char] st m [Char]
 anyLineNewline = (++ "\n") <$> anyLine
 
+-- | Parse indent by specified number of spaces (or equiv. tabs)
+indentWith :: Stream [Char] m Char
+           => HasReaderOptions st
+           => Int -> ParserT [Char] st m [Char]
+indentWith num = do
+  tabStop <- getOption readerTabStop
+  if (num < tabStop)
+     then count num (char ' ')
+     else choice [ try (count num (char ' '))
+                 , try (char '\t' >> count (num - tabStop) (char ' ')) ]
```

Comment:
What if you have, say, two tabs in a row; `indentWith 8` will fail, but should succeed, right?